packages/components/webpack.config.js

```diff
@@ -45,7 +45,17 @@ module.exports = function () {
                         {
                             loader: 'svgo-loader',
                             options: {
-                                plugins: [{ removeUselessStrokeAndFill: false }, { removeUnknownsAndDefaults: false }],
+                                plugins: [{
+                                    name: 'removeUselessStrokeAndFill',
+                                    params: {
+                                        attrs: 'false'
+                                        }
```

Comment:
looks like there is some formatting issue here

Reply:
Formatted them in the last commit.